### ISSUE TITLE
invalid configuration: kind and apiVersion is mandatory information that needs to be specified in all YAML documents

### ISSUE THREAD

**Issue by mayee007** [oracle/vagrant-boxes#114]
invalid configuration: kind and apiVersion is mandatory information that needs to be specified in all YAML documents

- created master node using vagrant 
- logged in to master 
- "su -"  
- /vagrant/scripts/kubeadm-setup-master.sh
   It asked for Oracle ID/pw, all good. 
  ....... 
Login Succeeded
./kubeadm-setup-master.sh: Setup Master node -- be patient!
./kubeadm-setup-master.sh: kubeadm-setup.sh did not complete successfully
Last lines of kubeadm-setup.log:
Checking br_netfilter module ...
Checking sysctl variables ...
Enabling kubelet ...
Created symlink from /etc/systemd/system/multi-user.target.wants/kubelet.service to /etc/systemd/system/kubelet.service.
Check successful, ready to run 'up' command ...
Waiting for kubeadm to setup master cluster...
Please wait ...

[ERROR] kubeadm init failed
invalid configuration: kind and apiVersion is mandatory information that needs to be specified in all YAML documents

**Comment by AmedeeBulle** [comment]
Apologies for this.

We have merged a fix (#110) to support Kubernetes 1.12.6 which should have been published by now. Unfortunately the release has been delayed due to a new K8s CVE.

We haven't reverted the this fix as  Kubernetes 1.12.7 should be released any time soon.

I just added the `k8s-1.12.5` branch in the repo before the #110 merge, if you refresh your local copy you can switch to this branch with
```
git checkout k8s-1.12.5
```
alternatively if you start from scratch you can directly clone that branch:
```
git clone https://github.com/oracle/vagrant-boxes.git --branch k8s-1.12.5
```

Let us know if this works for you

**Comment by mayee007** [comment]
Superb, it worked like a charm. Thanks for quick response.
